pyproject.toml

```diff
@@ -19,7 +19,8 @@ dependencies = [
     "appdirs",
     "pandas",  # remove!!!
     "python-dateutil",
-    "universal_pathlib"
+    "universal_pathlib",
+    "postgrest"
```

Comment:
I'm pretty sure you'll need the full supabase-py here, right now.

Strictly speaking, we'll only need gotrue and postgrest, but that's another refactor.

Reply:
You'll need to copy these two lines: https://github.com/laminlabs/lnhub-rest/blob/2dba53b7696fdf17d1e422a7bcb57a68a02d7990/pyproject.toml#L15-L16